gui/main.cxx

```diff
@@ -52,13 +56,42 @@ int main(int argc, char** argv)
   QApplication app(argc, argv);
   qtUtil::setApplicationIcon("mapgui");
 
+  bool help;
+  std::string openFilePath;
+
+  typedef kwiversys::CommandLineArguments argT;
+
+  argT arg;
+
+  arg.Initialize(argc, argv);
+
+  arg.AddBooleanArgument("--help", &help, "Display usage information");
+  arg.AddBooleanArgument("-h", &help, "Display usage information");
+  arg.AddArgument("--input", argT::SPACE_ARGUMENT,
+                  &openFilePath, "File to open at launch");
+  arg.AddArgument("-i", argT::SPACE_ARGUMENT,
+                  &openFilePath, "File to open at launch");
+
+  if(!arg.Parse() || help)
+  {
+    std::cout << "USAGE:" << std::endl;
+    std::cout << arg.GetHelp() << std::endl;
+    return EXIT_FAILURE;
+  }
+
   // Load Vital/MAP-Tk plugins
   auto const exeDir = QDir(QApplication::applicationDirPath());
   auto const rel_path = stdString(exeDir.absoluteFilePath("..")) + "/lib/maptk";
   kwiver::vital::algorithm_plugin_manager::instance().add_search_path(rel_path);
   kwiver::vital::algorithm_plugin_manager::instance().register_plugins();
 
   MainWindow window;
+
+  if (!openFilePath.empty())
+  {
+    window.openFile(QString::fromStdString(openFilePath));
```

Comment:
Please use `qtString` from `qtStlUtil.h`. It's... unclear if `QString::fromStdString` DTRT with respect to locales. (Although this will be irrelevant with `qtCliArgs`.)